Docs audit for credentials section (#317)
KILTprotocol/docs: docs/concepts/05_credentials/03_claiming.md

---

import CodeBlock from '@theme/CodeBlock';
import Claim from '@site/scripts/out/claim.json.raw!=!raw-loader!@site/scripts/out/claim.json';

As KILT is an open system, entities can make claims about any other entities, including themselves.
An entity can only trust a claim (as in the real world) if another trusted entity (called Attesters) *certifies* this claim.

> <!-- TODO: But what's a verifier? -->

Reviewer: **Claimers** are the ones that want some information about themselves certified. As part of our flow, they also issue the credentials (but these are invalid without an attestation.) Like filling up a "cType form".
**Attester** are the ones that check the truthfulness of your claims and certify them. Like give you a fancy stamp.
**Verifiers** are the ones that accept the credentials, they only verified that your certification is legit. Like just checking who did your stamp.

A verifier is like the guy who checks your ID to sell you alcohol, and he trust the government credentials only, not your bookclub saying you are 21.

Therefore, Verifiers might trust different Attesters for distinct scenarios.

## Creating a claim

In KILT, claims are based on claim types (CTypes).
Given a CType, a Claimer only needs to create a claim with the properties specified in the CType schema.

> The resulting claim contains a reference to the CType by its hash and includes the identity of the claim subject (identified by the `owner` property).

Reviewer: Maybe: 
```suggestion
The resulting claim contains a reference to the CType by its hash and includes the identity of the claim subject (identified by the `owner` property, whose value is a Kilt DID).
```

Author: ```suggestion
The resulting claim contains a reference to the CType by its hash and includes the identity of the claim subject (identified by the `owner` property, which has the value of a KILT DID).
```

<CodeBlock className="language-json" title="Claim example">
  {Claim}
</CodeBlock>

## Requesting a credential

> Once the Claimer has created a claim, and Attester needs to certify it and the resulting `Credential` sent to the chosen Attester using any messaging system.

Reviewer: I'm having a bit of trouble understanding this sentence. Maybe it needs some rewording. 
```suggestion
Once the Claimer has wrapped his claims into a `Credential`, it must be sent the chosen Attester using any messaging system to get them **certified**, i.e. attested.
```
